studiopress/genesis-custom-blocks
# Fix the issue of submenus not appearing

#### Changes
* Ensures that on first activation, the block CPT menus appear:
![the-submenu-items-appear](https://user-images.githubusercontent.com/4063887/96374793-97f31e00-113a-11eb-9cc2-4aa688a99528.png)

* This bug only appeared the first time GCB is *ever* activated on the site. 
#### Testing instructions
* Please see the steps to reproduce in BHH-554
* To destroy the site and create a new one, I've been doing:
```
wp db drop && wp db create && wp core install --url=https://foo.test --title="Example" --admin_user=foo --admin_email=example@example.com --admin_password=baz
```
